src/reducers/pieces.js

```diff
@@ -88,6 +118,13 @@ const pieces = (pieces = piecesDefault, action) => {
                 byId: {...pieces.byId, [action.id]: piece(pieces.byId[action.id], action)}
             };
         default:
+            /*init default empty value*/
```

Comment:
Drop this. Default action in redux shouldn't do anything. Instead use `pieces.byId && pieces.byId[...]` checks where needed